cg/store/database.py

```diff
@@ -0,0 +1,58 @@
+from typing import List, Optional
+
+from sqlalchemy import create_engine, inspect
+from sqlalchemy.engine.base import Engine
+from sqlalchemy.engine.reflection import Inspector
+from sqlalchemy.orm import Session, scoped_session, sessionmaker
+
+from cg.exc import CgError
+from cg.store.models import Model
+
+SESSION: Optional[scoped_session] = None
+ENGINE: Optional[Engine] = None
+
+
+def initialize_database(db_uri: str) -> None:
+    """Initialize the SQLAlchemy engine and session for status db."""
+    global SESSION, ENGINE
+    ENGINE = create_engine(db_uri, pool_pre_ping=True)
+    session_factory = sessionmaker(ENGINE)
+    SESSION = scoped_session(session_factory)
+
+
+def get_session() -> scoped_session:
+    """Get a SQLAlchemy session with a connection to status db."""
+    if not SESSION:
+        raise CgError("Database not initialised")
+    return SESSION
+
+
+def get_scoped_session_registry() -> Optional[scoped_session]:
+    """Get the scoped session registry for status db."""
+    return SESSION
```

Comment:
Is there a reason for keeping this function since it will return the same as `get_session`?

Reply:
The return types differ, I guess the other could be used and the exception caught. But I think I prefer this for now